azure/terraform/activitylogalerts.rego

```diff
@@ -22,6 +22,7 @@ azure_issue["azure_monitor_activity_log_alert_enabled"] {
 }
 
 azure_monitor_activity_log_alert_enabled {
+    lower(input.resources[_].type) == "azurerm_monitor_activity_log_alert"
     azure_attribute_absence["azure_monitor_activity_log_alert_enabled"]
     not azure_issue["azure_monitor_activity_log_alert_enabled"]
 }
```

Comment:
@rezoan there is no need for ```lower(input.resources[_].type) == "azurerm_redis_cache"```
because ```azure_attribute_absence``` will check that

Reply:
there was a situation where we need to handle 2 types of resource (one parent and one child). for that type of case we need to check the parent type explicitly in calling rule as azure_attribute_absence and azure_issue will check the child type.
But for single resource type you are correct.
Checking the type for single resource would nor hurt but i will remove those checking for single type resoruce in next PR.